pippo-session-parent/pippo-session-hazelcast/src/test/java/ro/pippo/session/hazelcast/HazelcastSessionDataStorageTest.java

```diff
@@ -65,18 +65,19 @@ public void testCreate() {
      */
     @Test
     public void testSave() {
-        HazelcastInstance hazelcastInstance2 = Hazelcast.newHazelcastInstance();
+//        HazelcastInstance hazelcastInstance2 = Hazelcast.newHazelcastInstance();
```

Comment:
@barrantesgerman Thanks. I don't know why `testSave` on this PR (in my opinion is nothing wrong in this). Please see first if that test fail on master branch.